tests/queries/0_stateless/03148_async_queries_in_query_log_errors.sh

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/env bash
+
+CUR_DIR=$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)
+# shellcheck source=../shell_config.sh
+. "$CUR_DIR"/../shell_config.sh
+
+function print_flush_query_logs()
+{
+    ${CLICKHOUSE_CLIENT} -q "SYSTEM FLUSH LOGS"
+    echo ""
+
+    echo "system.asynchronous_insert_log"
+        ${CLICKHOUSE_CLIENT} -q "
+          SELECT
+              database,
+              table,
+              query,
+              format,
+              extract(replace(exception::String, 'Exception', 'Exc*****on'), '.*UInt32:') as error,
+              not empty(flush_query_id) as populated_flush_query_id
+          FROM system.asynchronous_insert_log
+          WHERE
+              event_date >= yesterday()
```

Comment:
It's probably better to use `now() - INTERVAL 24 HOUR` otherwise the tests will flake when run at midnight...

Reply:
Notice it's `>= yesterday()` (the **>=** is the important part). So it's always covering at least 24 hours, normally more.

Reply:
Don't worry. You can find the same condition in many other tests, so I'm used to writing and seeing it.